index.html

```diff
@@ -0,0 +1,22 @@
+<html>
+  <body>
+    <script>
+      (async () => {
```

Comment:
I think we can reduce the cost of maintenance here by having this hosted at a central place so that we don't need to update all repos once we want to update this.

Also, not 100% sure whether we actually enforce the `v` prefix?